src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
```

Comment:
According to the checklist, we should not create several instances of the same class if we can use only one instance for our purposes. Here, you are creating a new instance of `Random` every time the `getRandomBall` method is called. It would be better to have one instance of `Random` as a field in the `Lottery` class and use it in this method.